### PR TITLE
add ROS to main.yml

### PR DESCRIPTION
**Pull request by kgaikwad**

### CODE REVIEW

File: main.yml [hunk 1]
```diff
@@ -741,3 +742,19 @@ vulnerability:
       - id: systems
         title: Systems
   source_repo: https://github.com/RedHatInsights/vulnerability-ui
+
+ros:
```
Comment (kruai) [inline main.yml]: I want to keep this config as alphabetized as possible. Can you please move this new config to be between `rhel-sw` and `ruledev`?

**Review by kruai — CHANGES_REQUESTED** [review]

File: main.yml (hunk 1 above)
Comment (kgaikwad) [reply main.yml]: surely, fixed the order and indentation as well.

File: main.yml [hunk 2]
```diff
@@ -604,6 +605,22 @@ rhel-sw:
       - id: x86
         title: x86
 
+ros:
+  title: Resource Optimization
+  api:
+    versions:
```
Comment (kgaikwad) [inline main.yml]: @kruai, as of now there is no API version released so will it be possible to add changes without this `versions` value? or is it mandatory to have?

File: main.yml [hunk 3]
```diff
@@ -604,6 +605,21 @@ rhel-sw:
       - id: x86
         title: x86
 
+ros:
+  title: Resource Optimization
+  api:
```
Comment (karelhala) [inline main.yml]: You don't have to include the api part at all if it's not ready. The API team should share api spec on `/${YOUR_APP_NAME}/${API_VERSION}/openapi.json` to allow customers to work with it. If it's not ready yet, you don't have to add it here and wait for it to be ready, once that you can open new PR to include it.

Comment (kgaikwad) [reply main.yml]: Great! As per a suggestion, removed api section. Thanks @karelhala.

File: main.yml (hunk 2 above)
Comment (kruai) [reply main.yml]: Looks like Karel answered this for you below, so I'm resolving :)

**Review by kruai — APPROVED** [review]

**Pull request merged**